<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>127</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stretch</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit1">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="btn1">
     <property name="text">
      <string>Get length</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>New length - mm:</string>
     </property>
     <property name="buddy">
      <cstring>edit1</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSlider" name="slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
